README.md

```diff
@@ -83,6 +83,7 @@ The **Maintained?** column shows:
 | [STMicroelectronics b2260 - h410 (96boards fmt)](http://www.st.com/web/en/catalog/mmc/FM131/SC999/SS1628/PF258776) |`PLATFORM=stm-b2260`| No | ![Actively maintained](documentation/images/green.svg) |
 | [STMicroelectronics b2120 - h310 / h410](http://www.st.com/web/en/catalog/mmc/FM131/SC999/SS1628/PF258776) |`PLATFORM=stm-cannes`| No | ![Actively maintained](documentation/images/green.svg) |
 | STMicroelectronics stm32mp1 |`PLATFORM=stm32mp1`| No | ![Actively maintained](documentation/images/green.svg) |
+| [Allwinner A64 Pine64 Board](https://www.pine64.org/) |`PLATFORM=sunxi-sun50i_a64`| Yes | ![Actively Maintained](documentation/images/green.svg) |
```

Comment:
s/Allwinner A64 Pine64 Board/Pine64 A64 board/

About the `PLATFORM_FLAVOR`: isn't `sun50i_a64` too generic/too obscure? Why not `pine64_a64`?

Reply:
Actually, all the 64 bits SoCs from allwinner has grouped with SUN50I prefix, so idea is to have
sun50i_a64 ,sun50i_h6 and sun50i_h64 etc as PLATFORM_FLAVOR .

Reply:
OK, fair enough.